app/controllers/app.py

```diff
@@ -1224,3 +1224,70 @@ def post(self, project_id, app_id):
             return dict(status='fail', message='No values found'), 404
 
         return dict(status='success', data=dict(values=cpu_data_list)), 200
+
+
+class AppNetworkUsageView(Resource):
+    @jwt_required
+    def post(self, project_id, app_id):
+
+        current_user_id = get_jwt_identity()
+        current_user_roles = get_jwt_claims()['roles']
+
+        app_memory_schema = MetricsSchema()
```

Comment:
@Mubangizi it will become quite confusing later if we have this named as `app_memory_schema` in a route for network

Reply:
Thank you for the observation, I have fixed this